projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -4136,6 +4136,51 @@ def ElementwiseBitwiseAndScalarInt8Module_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ElementwiseQuantizePerTensorModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float, True),
+    ])
+    def forward(self, x):
+        scale = 0.04
+        zp = -110
+        dtype = torch.qint8
+        q = torch.quantize_per_tensor(x, scale, zp, dtype)
+        return q.int_repr()
```

Comment:
IIUC this `int_repr` is load bearing to the test because we don't support returning `qint*` types; maybe worth adding a note? It's clearer from the lowering for dequantization.

Reply:
Added a comment - and yes, its because IREE does not yet support adding the sidechannel information required for `quint*` types.